charts/psmdb-operator/values.yaml

```diff
@@ -9,6 +9,13 @@ image:
   tag: 1.14.0
   pullPolicy: IfNotPresent
 
+# disableTelemetry: according to
+# https://docs.percona.com/percona-operator-for-mongodb/telemetry.html
+# this is how you can disable telemetry collection
+# default is false which means telemetry will be collected
+# NOTE: "true" or "false" must be a string
+disableTelemetry: "false"
```

Comment:
Let's remove the quotes and a note about a string like in a PXC chart: https://github.com/percona/percona-helm-charts/blob/main/charts/pxc-operator/values.yaml#L56

Reply:
done